src/lsdb/nested/datasets/generation.py

```diff
@@ -0,0 +1,74 @@
+from nested_pandas import datasets
+
+import lsdb.nested as nd
+
+
+def generate_data(n_base, n_layer, npartitions=1, seed=None) -> nd.NestedFrame:
```

Comment:
I'd really like this to generate data with radec values, so it can be made into a catalog.

But again - can be in a follow-up.

Reply:
Will make an issue!